lua/lvim/plugin-loader.lua

```diff
@@ -106,14 +106,20 @@ function plugin_loader.load(configurations)
         end
       end
     end)
-    -- colorscheme must get called after plugins are loaded or it will break new installs.
-    vim.g.colors_name = lvim.colorscheme
-    vim.cmd("colorscheme " .. lvim.colorscheme)
   end, debug.traceback)
+
   if not status_ok then
     Log:warn "problems detected while loading plugins' configurations"
     Log:trace(debug.traceback())
   end
+
+  -- colorscheme must get called after plugins are loaded or it will break new installs.
+  vim.g.colors_name = lvim.colorscheme
+  local set_colorscheme_ok, _ = pcall(vim.cmd, "colorscheme " .. lvim.colorscheme)
+
+  if not set_colorscheme_ok then
+    Log:warn(string.format("problem detected: could not find '%s' colorscheme", lvim.colorscheme))
+  end
```

Comment:
```suggestion
  -- ref: https://github.com/neovim/neovim/issues/18201#issuecomment-1104754564
  local colors = vim.api.nvim_get_runtime_file(("colors/%s*"):format(lvim.colorscheme), true)
  if #colors == 0 then
    Log:warn(string.format("problem detected: could not find '%s' colorscheme", lvim.colorscheme))
    return
  end
  vim.g.colors_name = lvim.colorscheme
  vim.cmd("colorscheme " .. lvim.colorscheme)
```

Reply:
We might not need to pass `true`? if there's one, that's enough, we don't need to keep searching

For reference
```lua
nvim_get_runtime_file({name}, {all})                 *nvim_get_runtime_file()*
    Find files in runtime directories

    'name' can contain wildcards. For example
    nvim_get_runtime_file("colors/*.vim", true) will return all color scheme
    files.

    It is not an error to not find any files. An empty array is returned then.

    Parameters:  
        {name}  pattern of files to search for
        {all}   whether to return all matches or only the first
```

Reply:
I wonder why we need to set `vim.g.colors_name`. When most colorschemes will do it for us
https://github.com/LunarVim/LunarVim/blob/6ab3e8a73920e3d2e02e57f7dd3a1f3d8e54ee63/lua/lvim/plugin-loader.lua#L110